practice-app/back-end/routes/twittercomments.js

```diff
@@ -2,8 +2,47 @@ var express = require('express');
 var router = express.Router();
 
 /* GET twitter comments. */
-router.get('/', function(req, res, next) {
-  res.send('respond');
-});
+router.get('/', function (req, res, next) {
+  // import twitter
+  var Twitter = require('twitter');
+  //get the all access keys
+  var key = require('./key.json');
+  //create new twitter object
+  var client = new Twitter(key);
+
+  //To specify searching parameters
+  var params = {
+    q: 'github',
```

Comment:
you have to get params from  req 

So this should work:
curl GET 'http://localhost:3000/locations?query=3&count=100'


So in this version it only searches   'github'